packages/as-proba/assembly/probability/sampler.ts

```diff
@@ -101,7 +117,10 @@ export class Sampler {
   }
 
   /**
-   * Populate observation zone bounderies.
+   * This method populates observation zone bounderies.
+   *
+   * @remarks
+   * - This method is used by the {@link inverseCumulativeDistributionSampling} method.
```

Comment:
Why is it used by that other function (I mean i don't fully understand probabilities that might be obvious but maybe we want to explain ?)

Reply:
it would amount to repeating myself with its description

Reply:
Your'e right, and this method is private so ok for me